Scenarios/ITU-R_M2412-0_5BA.m

```diff
@@ -0,0 +1,50 @@
+clear all 
+
+% Disable cast to struct warnings
+w = warning ('off','all');
+
+%% Get configuration
+Config = MonsterConfig(); % Get template config parameters
+
+%add scenario specific setup for 'ITU-R M2412-0 5.B.A':
+% from https://www.itu.int/dms_pub/itu-r/opb/rep/R-REP-M.2412-2017-PDF-E.pdf Table 5.c Configuration A
+    %For Spectral efficiency and mobility Evaluations.
+    Config.Scenario = 'ITU-R M.2412-0 5.B.A';
+    Config.MacroEnb.number = 19;
+    Config.MicroEnb.number = 0;
+    Config.PicoEnb.number = 0;				
+    Config.Phy.downlinkFrequency = 4000; %MHz
+    Config.MacroEnb.height= 25;
+    Config.MacroEnb.Pmax = 10^(41/10)/1e3; %41dBm converted to W
+    Config.MacroEnb.subframes = 50; % 10MHz bandwidth
+    %Ue Transmit power in dBm = 23. 
+    %Percentage of high loss and low loss: 20/80 (high/low)
+    Config.MacroEnb.radius = 200; %intersite distance in meters
```

Comment:
Oh yeah, quick question; is this the intersite distance or the cell radius? From what I've been seeing it seems like it's the cell radius

Reply:
It is another case of wrong naming. It is used as the intersite distance, though it is called radius. I'll fix this :)